app/Helpers/PBOMission/Mission/SQMConfig.php

```diff
@@ -128,7 +128,7 @@ private function parseLine($line) {
 
     // Read end of class/attribute
     if ($line == '}') {
-      if ($this->mode == SQM_READ_MODE_CLASS) {
+      if ($this->mode == "class" || $this->mode == SQM_READ_MODE_CLASS) {
```

Comment:
The parser has 3 modes: SQM_READ_MODE_CLASS (0), SQM_READ_MODE_ATTRIBUTE (1), and "class"
The SQM_READ_MODE_x aren't enums because php7.4 doesn't have enums, so they are just constants equal to 0/1

php8 changes how strings are compared to numbers, so previously:
0 == "class" == SQM_READ_MODE_CLASS
but now:
0 != "class" != SQM_READ_MODE_CLASS

$this->mode == SQM_READ_MODE_CLASS was used as a dirty way of checking if the mode is either 0 or "class"

![image](https://user-images.githubusercontent.com/28692063/168706971-21ac5c3d-98d0-4a79-bc85-95373458d42b.png)